gtk4/src/auto/enums.rs

```diff
@@ -1679,8 +1684,10 @@ pub enum Align {
     End,
     #[doc(alias = "GTK_ALIGN_CENTER")]
     Center,
-    #[doc(alias = "GTK_ALIGN_BASELINE")]
-    Baseline,
```

Comment:
This is a bit problematic, as it maps to the same value as `BaselineFill`. I guess we will have to manually write this trait so we can conditionally either have `Baseline` or `BaselineFill`

Reply:
Fixed

Reply:
In a later commit. Makes sense!